### PR TITLE
Restructure end-user section in /community

### PR DESCRIPTION
**Pull request by svrnm**
I thought it would look better in the /community if the content for the end user forums is in sub-pages, let me know what you think

https://deploy-preview-1872--opentelemetry.netlify.app/community/end-user/

### CODE REVIEW

File: content/en/community/end-user/_index.md [hunk 1]
```diff
@@ -5,78 +5,33 @@ Description: Vendor-agnostic resources for users of OpenTelemetry
...
-excited to announce two new ways for you to connect with other users of the
-project and share best practices, a synchronous monthly discussion group and a
-new private slack channel. These forums will bring together operations and
+desire a vendor-agnostic space to discuss adopting OpenTelemetry, so we provide
+multiple ways for you to connect with other users of the project and share best
+practices:
+
+* [A synchronous monthly discussion group](/community/end-user/discussion-group/)
+* [A private slack channel](/community/end-user/slack-channel/)
+* [Talks about OTel in practice](/community/end-user/otel-in-practice/)
```
Comment (melbgirl) [inline content/en/community/end-user/_index.md:14]: Would be good for these to be links, eg to the Slack channel itself.

**Review by melbgirl — COMMENTED** [review]
Could you add a link to the netlify preview please? It’s hard to visualize the experience from the file changes.

**Comment by svrnm** [comment]
> Could you add a link to the netlify preview please? It’s hard to visualize the experience from the file changes.

done

File: content/en/community/end-user/_index.md (hunk 1 above)
Comment (svrnm) [reply content/en/community/end-user/_index.md:14]: it's a private slack channel, so providing a link doesn't do the trick.

**Comment by melbgirl** [comment]
Thanks @svrnm - looks good to me.
One question: some of the content is similar to the content in the GitHub community repo https://github.com/open-telemetry/community/tree/main/working-groups/end-user. We should ensure that the content in each matches the other, and consider long-term whether it makes sense to keep both sources (ie website and repo files) for this info.

File: content/en/community/end-user/_index.md (hunk 1 above)
Comment (chalin) [inline content/en/community/end-user/_index.md:14]: You should be able to make these relative links:

```suggestion
* [A synchronous monthly discussion group](discussion-group/)
* [A private slack channel](slack-channel/)
* [Talks about OTel in practice](otel-in-practice/)
```

**Review by chalin — COMMENTED** [review]
See (minor) inline comment.

File: content/en/community/end-user/discussion-group.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+---
+title: Monthly Discussion Group
```
Comment (chalin) [inline content/en/community/end-user/discussion-group.md:2]:
```suggestion
title: Monthly Discussion Group
linkTitle: Monthly Discussions
```

**Comment by svrnm** [comment]
> @svrnm I can create a PR for the above if you like. I think the new update looks good so far! My only other suggestion right meow is to add a space for 'latest news' but I can also add in a PR for that following this new format.

@reese-lee Yes, let's have follow up PRs, the intend of this PR was just to restructure some things. 
Can you approve the PR on behalf of the end user WG then we merge it in

**Review by sharrmander — APPROVED** [review]
Thanks for adding this

**Review by chalin — APPROVED** [review]

**Pull request merged**